src/azure-cli/azure/cli/command_modules/apim/_help.py

```diff
@@ -653,3 +653,75 @@
     text: |
         az apim deletedservice purge --service-name MyApim --location westus
 """
+
+helps['apim graphqlapi resolver create'] = """
```

Comment:
Could we make the naming of command more concise?
```suggestion
helps['apim graphql resolver create'] = """
```